templates/web-api-svc.yaml

```diff
@@ -26,11 +26,11 @@ spec:
     - {{ . }}
     {{- end }}
   {{- end }}
-  {{- if and (eq "ClusterIP" .Values.web.service.api.type) .Values.web.service.api.clusterIP }}
-  clusterIP: {{ .Values.web.service.api.clusterIP }}
+  {{- if and (eq "ClusterIP" .Values.web.service.api.type) (ne "None" .Values.web.service.api.clusterIP) }}
```

Comment:
I'm not sure this is quite what we want. `None` is actually a special value that indicates the service should be headless. I'm not sure why we made it the default, but it's clearly causing problems, and I don't think we need these services to be headless.

I think an approach similar to #163 is preferable, which changes the default to `nil`. Let me know if you disagree

Reply:
Thank-you @aoldershaw.  Yes, that makes sense; apologies; I'm new to this whole tech stack and had assumed that None was just another null / nil / nothing alternative.  I've now read up and understand that:
- `None` = "Don't use 1 IP for this group of pods, give me all the members' IPs" 
- `nil` = "I don't care what the specific IP for this group is; assign it any free one"